doc/esp8266wifi/station-class.rst

```diff
@@ -122,16 +122,16 @@ The following IP configuration may be provided:
    (like e.g. *www.google.co.uk*) and translate them for us to IP
    addresses
 
-For Arduino networking API compatibilty the ESP8266WiFi library supports for IPv4 additional versions of the ``config`` function.
+For Arduino networking API compatibility, the ESP8266WiFi library supports IPv4-only additional versions of the ``config`` function:
 
 .. code:: cpp
 
-    WiFi.config(local_ip)
-    WiFi.config(local_ip, dns)
-    WiFi.config(local_ip, dns, gateway)
+    WiFi.config(local_ip)                        (discouraged, for retro compatibility: dns=gateway=···.1)
+    WiFi.config(local_ip, dns)                   (discouraged, for retro compatibility: gateway=···.1)
+    WiFi.config(local_ip, dns, gateway)          (discouraged, for retro compatibility: subnet/netmask quite unavoidable)
```

Comment:
"subnet/netmask quite unavoidable" is a bit unclear.
Maybe better: "Assumes subnet mask of 255.255.255.0"

And "retro compatibility" with what?
I know it is retro Arduino compatibility.
But maybe that's a bit too long to repeat?